x/tokenfactory/keeper/msg_server.go

```diff
@@ -95,6 +97,12 @@ func (server msgServer) Burn(goCtx context.Context, msg *types.MsgBurn) (*types.
 		msg.BurnFromAddress = msg.Sender
 	}
 
+	accountI := server.Keeper.accountKeeper.GetAccount(ctx, sdk.AccAddress(msg.BurnFromAddress))
+	_, ok := accountI.(authtypes.ModuleAccountI)
+	if ok {
+		return nil, types.ErrBurnFromModuleAccount
+	}
+
```

Comment:
Can someone confirm this newly added logic for checking if the burn from account is not an module account before merging? @ValarDragon @czarcas7ic

Reply:
This make sense to me, but is there any reason tokenfactory does not have a msg_server_test? It appears other modules have this

Reply:
@p0mvn Would you be able to check this one logic? Thanks